src/Cities/App/Controllers/ListCityController.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Cities\App\Controllers;
+
+use Illuminate\Http\JsonResponse;
+use Illuminate\Http\Request;
+use Lightit\Cities\App\Transformers\CityTransformer;
+use Lightit\Cities\Domain\Actions\ListCityAction;
+
+class ListCityController
```

Comment:
Following naming conventions, this would be Index instead of list

Reply:
Where can I find the documentation for those naming conventions? I couldn't find it on Slite

I followed the naming used in our laravel project structure standards repo
https://github.com/Light-it-labs/laravel/tree/11.x/src/Backoffice/Users/App/Controllers

Reply:
My comment was following Laravel's standard https://laravel.com/docs/11.x/controllers#actions-handled-by-resource-controllers but you are right we are not following it on the boilerplate. Dont worry about this comment, I'll discuss it with Sergio and TLs

Reply:
Ok thanks for the doc!